cmd/spire-server/cli/cli.go

```diff
@@ -12,10 +12,15 @@ import (
 	"github.com/spiffe/spire/cmd/spire-server/cli/run"
 	"github.com/spiffe/spire/cmd/spire-server/cli/token"
 	"github.com/spiffe/spire/cmd/spire-server/cli/x509"
+	"github.com/spiffe/spire/pkg/common/log"
 	"github.com/spiffe/spire/pkg/common/version"
 )
 
-func Run(args []string) int {
+type CLI struct {
+	logOptions []log.Option
```

Comment:
Should this be exported so it can be set by `package main`?

Reply:
Of course, that's the whole goal I had :)